app/js/views/element_call_button_view.js

```diff
@@ -0,0 +1,54 @@
+import { View } from 'components/fxos-mvc/dist/mvc';
+import 'components/gaia-button/gaia-button';
+
+(function(window) {
+'use strict';
+
+var shadowTemplate =
+`<style scoped>
+gaia-button {
+	--button-background: green;
+	--button-background-active: lightgreen;
+	--button-color: white;
+	--button-color-active: white;
+	width: 10rem;
+	min-width: 0 !important;
+	margin: 0.5rem !important;
+}
+</style>
+<gaia-button>Call</gaia-button>`;
+
+var proto;
+
+export default class ElementCallButtonView extends View {
+	constructor(controller, phoneFieldController) {
+		proto = Object.create(HTMLElement.prototype);
+		proto.createdCallback = this._createdCallback;
+		proto._scope = this;
```

Comment:
I used FXOSCamera as a reference for building this originally, but I thought that we could break up the logic further. I agree though that using custom elements as a view isn't going to work. I'm still not really satisfied with this approach either, though.